src/fire/logoutFire.js

```diff
@@ -31,8 +31,19 @@ function logoutFirebase(event) {
 
 // Handle Account Status
 // I don't know why this makes it work but is does
+
+// not sure if I should move this out of this file ?
```

Comment:
Yeah lets move this into a different file since that's what we've been doing for other js functions.